assembl/models/preferences.py

```diff
@@ -288,14 +288,29 @@ def validate_single_value(self, key, value, pref_data, data_type):
             elif data_type == "scalar":
                 assert value in pref_data.get("scalar_values", ()), (
                     "value not allowed: " + value)
+            elif data_type == "address":
+                condition = False
+                while not condition:
+                    if not value.strip():
+                        # No empty strings allowed
+                        break
+                    # Whilst not an address, requested feature
+                    if value in ("*",):
+                        condition = True
+                        break
+                    elif is_valid_ipv4_address(value):
+                        condition = True
+                        break
+                    elif is_valid_ipv6_address(value):
+                        condition = True
+                        break
+                    else:
+                        condition = bool(urlparse(value).scheme)
```

Comment:
Same comment: Address and URL are distinct types. I'm fine if you accept a domain at this line, not a URL.